gateway/.dockerignore

```diff
@@ -26,3 +26,4 @@ web/static/CACHE
 stats
 Dockerfile
 *.sqlite3
+.tox/
```

Comment:
Could you add `__pycache__` here too? 🙏

Reply:
it's already at https://github.com/Qiskit-Extensions/quantum-serverless/blob/e64024d5c0f1163063421a4d6d6b8a77161a7ee7/gateway/.dockerignore#L17

Reply:
True! Thanks, Paul!